src/Microsoft.Framework.SecurityHelper.Sources/SecurityHelper.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+using System.Security.Claims;
+
+namespace Microsoft.Framework.Internal
+{
+    /// <summary>
+    /// Helper code used when implementing authentication middleware
+    /// </summary>
+    internal static class SecurityHelper
+    {
+        /// <summary>
+        /// Add all ClaimsIdenities from an additional ClaimPrincipal to the ClaimsPrincipal
```

Comment:
If you're going to take the trouble to write docs you can just use `<see cref="..." />`